src/components/ApplicationDetailsView/ComponentListItem.tsx

```diff
@@ -81,14 +78,18 @@ export const ComponentListItem: React.FC<ComponentListViewPageProps> = ({
                 label: 'Component settings',
               },
               {
-                cta: () => showLogsForComponent(component),
+                cta: () =>
+                  showModal(
+                    launchComponentLogViewerModal({
+                      component,
+                      title: 'View build logs',
+                    }),
+                  ),
```

Comment:
```suggestion
                  showModal(
                    launchComponentLogViewerModal({
                      component,
                      title: 'View build logs',
                      className: "hacDev-log-viewer-modal",
                      variant: ModalVariant.large
                    }),
                  ),
```

Reply:
I figured another way to pass these, I moved the Title inside the `BuildLogViewer` component that is rendered inside the Modal
and the remaining props that are needed for the modal I am now passing those as `args` along with the Component while calling the `createModalLauncher`. Let me know what you think about this